src/cljs/hatti/views/dataview.cljs

```diff
@@ -95,8 +98,8 @@
                        (if (and (= view :map) no-geodata?)
                          [:a {:class "inactive" :title "No geodata"}
                           (name view)]
-                         [:a {:href (str "#/" (name view))
-                              :class (view->cls view)} label]))]
+                         [:a {:on-click (click-fn #(activate-view! view))
+                              :href "#" :class (view->cls view)} label]))]
```

Comment:
Yup, we need to revert this change, since we want nav changes to reflect in the URL, and secretary to handle changes off of that.